src/dgp/regression.py

```diff
@@ -3,11 +3,11 @@
 
 import jax
 import jax.numpy as jnp
-from jaxtyping import Array, Float, PyTree
+from jaxtyping import Array, Float
 
-from dgp.kernels import derivative_cov_func, CovMatrix
+from dgp.kernels import CovMatrix, derivative_cov_func
 
-_jitter = 1e-6
+_jitter = 1e-3
```

Comment:
This is quite a large default jitter, which I don't think is a good idea. A default jitter of 1e-6 is in line with other GP libraries, but it would make sense to be able to specify the exact jitter as an argument to the relevant functions. How about that approach?

Reply:
Yep it should be a variable; updated.